README.md

```diff
@@ -1,296 +1,92 @@
-
-**Note:** If someone else has made changes to the PDS member (or PS) while you were editing it, you can merge your conflicts before uploading to the mainframe.
-
-![Edit](/docs/images/ZE-download-edit.gif?raw=true "Edit")
-<br /><br />
-
-### Use the save option to prevent merge conflicts
-
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Open a member of a data set you want to edit.
-4. Edit a data set.
-5. Press Ctrl+S or Command+S (OSx) to save you changes.
-6. (Optional) Resolve merge conflicts if necessary.
+- Configure TSO/E address space services, z/OS data set, file REST interface and z/OS jobs REST interface. For more information, see [z/OS Requirements](https://docs.zowe.org/stable/user-guide/systemrequirements-zosmf.html#z-os-requirements).
 
-![Save](/docs/images/ZE-safe-save.gif?raw=true "Save")
-<br /><br />
+## Directory Structure
 
-### Create a new PDS and a PDS member
+Zowe Explorer repository includes several folders with files that let you build and configure various aspects of the extension. The bulk of the Zowe Explorer source code is in the `packages` directory. The `packages` folder has the following structure:
 
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Click the **Create New Data Set** button to create a PDS.
-4. From the drop-down menu, select the type of PDS that you want to create.
-5. Enter a name for the PDS.
-   The PDS is created.
-6. To create a member, right-click the PDS and select **Create New Member**.
-7. Enter a name for the member.
-   The member is created.
+- `├──`[`packages`](./packages) — includes source code of Zowe Explorer, various Zowe Explorer-related extensions, and the extensibility API<br>
+  - `├──`[`eslint-plugin-zowe-explorer`](./packages/eslint-plugin-zowe-explorer) — includes necessary files to configure ESLint plug-in for Zowe Explorer
+  - `├──`[`zowe-explorer-api`](./packages/zowe-explorer-api) — includes files to set up and use Extensibility API for Explorer. The API has two modules: Profiles API and Tree API.
+  - `├──`[`zowe-explorer-ftp-extension`](./packages/zowe-explorer-ftp-extension) — includes files to set up and use the FTP extension for Zowe Explorer.
+  - `├──`[`zowe-explorer`](./packages/zowe-explorer) — core Zowe Explorer source files, ReadMe, Changelog, and more.
 
-![Create](/docs/images/ZE-cre-pds-member.gif?raw=true "Create")
-<br /><br />
+## Build Locally
 
-### Delete a PDS member and PDS
+Ensure that you meet the [software requirements](#requirements) before you build and test your Zowe Explorer.
 
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Open the profile and PDS containing the member.
-4. Right-click on the PDS member that you want to delete and select **Delete Member**.
-5. Confirm the deletion by clicking **Yes** on the drop-down menu.
+Clone the repository and run `yarn workspace vscode-extension-for-zowe package` to build a VSIX file and start working with the extension.
```

Comment:
I tried this command and I ended up with just one .vsix file. It is better to replace `yarn workspace vscode-extension-for-zowe package` to `yarn run package`. And after that, maybe  it is good to mention that the vsix is in the `dist` folder